src/main/java/data/Database.java

```diff
@@ -60,20 +58,28 @@ public OutputStream writeData(String runId, String type) {
     return null;
   }
 
-  /*Returns appropiate file name for data with this runId and type. */
-  private String FileName(String runId, String type) {
-    return getDirectoryName() + "/" + runId + "_" + type;
+  /* Returns appropiate file name for data with this runId and type. */
+  private String fileName(String runId, String type) {
+    return getDatabaseName() + "/" + runId + "_" + type;
   }
 
-  /* Adds file to database given its runid and type. */
-  public String newDatabaseEntry(String runId, String type) {
-    String fileName = FileName(runId, type);
-    return fileName;
+  /* Returns list of files in database. */
+  public ArrayList<String> getAllFiles() {
+    File[] files = folder.listFiles();
+    ArrayList<String> filesAsStrings = new ArrayList<String>();
+    for (int i = 0; i < files.length; i++) filesAsStrings.add(files[i].getName());
```

Comment:
Prefer to use for-each loops by default, instead of manually looping over indices.

https://docs.oracle.com/javase/8/docs/technotes/guides/language/foreach.html